docs/branch_based_planner/getting-started.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started With Branch Planner
```

Comment:
We'll have a "what Branch Planner is and why you should care" page, right? Connecting the pages would be a good idea; on this one, a line with the link would be enough.

Reply:
Could copy-paste much of what's here already: https://github.com/weaveworks/tf-controller/issues/527

Reply:
I don't know if we'll have a page or not, I assume yes, but I had no intention to add that into this document. I think a separate "Introduction to Branch Planner" doc can link to this doc as "next step".